components/inputs/docs/input-date-time.md

```diff
@@ -289,10 +381,11 @@ To make your usage of `d2l-input-date-time` accessible, use the following proper
 | Attribute | Description |
 |--|--|
 | `label` | **REQUIRED** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/) |
+| `label-hidden` | Use if label should be visually hidden but available for screen reader users |
 
 ## Date-Time Range Input [d2l-input-date-time-range]
 
-The `<d2l-input-date-time-range>` component consists of two input-date-time components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
+Use the `<d2l-input-date-time-range>` component when users need to enter a single date and time, like a due date. The component consists of two input-date-time components - one for the start of a range and one for the end of a range. Values specified for these components (through the `start-value` and/or `end-value` attributes) are displayed if specified, and selected values are reflected.
```

Comment:
Is this the correct text to go under Date-time range?

Reply:
I'll take a look!